src/renderer/containers/QuickStart/VerifyEmailStep.tsx

```diff
@@ -0,0 +1,138 @@
+import { colors } from "@common/colors";
+import { css } from "@emotion/react";
+import styled from "@emotion/styled";
+import CheckCircleOutlineIcon from "@mui/icons-material/CheckCircleOutline";
+import { Button } from "@mui/material";
+import Box from "@mui/material/Box";
+import React, { useEffect } from "react";
+
+import { ExternalLink as A } from "@/components/ExternalLink";
+import { useAccount } from "@/lib/hooks/useAccount";
+import { useServices } from "@/services";
+
+import { QuickStartHeader } from "./QuickStartHeader";
+
+const Container = styled.div`
+  margin: 0 auto;
+  width: 100%;
+  max-width: 800px;
+`;
+
+const classes = {
+  message: css`
+    color: ${colors.textSecondary};
+  `,
+  emailContainer: css`
+    background: ${colors.purpleDark};
+    padding: 10px 20px;
+    border-radius: 5px;
+    margin: 10px;
+    width: fit-content;
+    font-size: 20px;
+  `,
+  incorrectEmailContainer: css`
+    margin-top: -5px;
+    margin-left: 10px;
+    font-size: 14px;
+    color: ${colors.textDim};
+    > a {
+      color: ${colors.purplePrimary};
+    }
+  `,
+  instructions: css`
+    margin-top: 25px;
+    margin-bottom: 10px;
+    color: ${colors.textSecondary};
+  `,
+  emailNotFoundContainer: css`
+    margin-top: 4px;
+    font-size: 14px;
+    color: ${colors.textDim};
+    > a {
+      color: ${colors.purplePrimary};
+    }
+  `,
+  confirmationContainer: css`
+    margin-top: 30px;
+    font-size: 28px;
+    color: ${colors.greenPrimary};
+    display: grid;
+    align-items: center;
+    gap: 8px;
+    grid-template-columns: auto auto 1fr;
+    > svg {
+      font-size: 40px;
+    }
+  `,
+};
+
+export const VerifyEmailStep: React.FC = () => {
+  const { authService } = useServices();
+  const setServerError = useAccount((store) => store.setServerError);
+  const setUser = useAccount((store) => store.setUser);
+  const user = useAccount((store) => store.user);
+  const emailVerificationSent = useAccount((store) => store.emailVerificationSent);
+  const setEmailVerificationSent = useAccount((store) => store.setEmailVerificationSent);
+
+  const profileUrl = "https://slippi.gg/profile";
```

Comment:
```suggestion
  const profileUrl = "https://slippi.gg/profile";
```
Can we either declare this constant outside of the React component, or put the URL in `src/common/constants.ts`?